core/assets/bundles/bundle_ja.properties

```
@@ -1023,6 +1101,11 @@ unit.minke.name = ミンク
 unit.bryde.name = ブライド
 unit.sei.name = セイ
 unit.omura.name = オムラ
+unit.retusa.name = リトゥサ
```

CR: 
```suggestion
unit.retusa.name = レトゥーザ
```
Ficus Retusa が `Ficus ruh too zuh` と発音するらしいです。